packages/main/src/SegmentedButton.ts

```diff
@@ -179,7 +179,7 @@ class SegmentedButton extends UI5Element {
 		switch (this.mode) {
 		case SegmentedButtonMode.MultiSelect:
```

Comment:
This switch case might be replaced with if else overall.

Reply:
I think it will be better to refactor it in separate change as it is not related directly to the topic of this one.